.github/ISSUE_TEMPLATE/content-change-request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Content Change Request
+about: Suggest a change (typo correction, enhancement, removal, adjustment. etc.)
+  on any existing part of the documentation
+title: Content Change Request
+labels: ''
+assignees: ''
+
+---
+
+**Which part of the repository is your change request related to?**
+Please provide the exact URL(s) of the page(s) [...]
+
```

Comment:
I would add an additional point here. 
** Please describe  which content you would expect here **
Describe a little more in detail which information or resources you would expect here.